ratis-proto/src/main/proto/Raft.proto

```diff
@@ -409,13 +409,20 @@ message SetConfigurationRequestProto {
   enum Mode {
     SET_UNCONDITIONALLY = 0;
     ADD = 1;
+    CAS = 2;
   }
   RaftRpcRequestProto rpcRequest = 1;
-  repeated RaftPeerProto peers = 2;
-  repeated RaftPeerProto listeners = 3;
+  SetConfigurationSevers servers = 2;
```

Comment:
This is an incompatible change.  Let's simply add the new lists and use "currentPeers" for the current conf.
```
message SetConfigurationRequestProto {
  enum Mode {
    SET_UNCONDITIONALLY = 0;
    ADD = 1;
    COMPARE_AND_SET = 2;
  }
  RaftRpcRequestProto rpcRequest = 1;
  repeated RaftPeerProto peers = 2;
  repeated RaftPeerProto listeners = 3;
  optional Mode mode = 4;
  repeated RaftPeerProto currentPeers = 5;
  repeated RaftPeerProto currentListeners = 6;
}
```

Reply:
Good catch, I have fixed it